src/test/java/org/fundacionjala/coding/marcoslara/spinwords/SpinWordsTest.java

```diff
@@ -1,65 +0,0 @@
-package org.fundacionjala.coding.marcoslara.spinwords;
```

Comment:
Do you remember how to apply git reset ?
1. git reset HEAD~6
2. git commit again
3. git push --force